src/wizard/java/index.md

```diff
@@ -5,139 +5,53 @@ support_level: production
-    void unsafeMethod() {
-        throw new UnsupportedOperationException("You shouldn't call this!");
-    }
 
-    /**
-      * Examples using the (recommended) static API.
-      */
-    void logWithStaticAPI() {
-        // Note that all fields set on the context are optional. Context data is copied onto
-        // all future events in the current context (until the context is cleared).
-
-        // Record a breadcrumb in the current context. By default the last 100 breadcrumbs are kept.
-        Sentry.getContext().recordBreadcrumb(
-            new BreadcrumbBuilder().setMessage("User made an action").build()
-        );
-
-        // Set the user in the current context.
-        Sentry.getContext().setUser(
-            new UserBuilder().setEmail("hello@sentry.io").build()
-        );
-
-        // Add extra data to future events in this context.
-        Sentry.getContext().addExtra("extra", "thing");
-
-        // Add an additional tag to future events in this context.
-        Sentry.getContext().addTag("tagName", "tagValue");
-
-        /*
-         This sends a simple event to Sentry using the statically stored instance
-         that was created in the ``main`` method.
-         */
-        Sentry.capture("This is a test");
-
-        try {
-            unsafeMethod();
-        } catch (Exception e) {
-            // This sends an exception event to Sentry using the statically stored instance
-            // that was created in the ``main`` method.
-            Sentry.capture(e);
-        }
-    }
-
-    /**
-      * Examples that use the SentryClient instance directly.
-      */
-    void logWithInstanceAPI() {
-        // Retrieve the current context.
-        Context context = sentry.getContext();
-
-        // Record a breadcrumb in the current context. By default the last 100 breadcrumbs are kept.
-        context.recordBreadcrumb(new BreadcrumbBuilder().setMessage("User made an action").build());
+  public static void main(String... args) {
+    Sentry.init(options -> {
+      options.setDsn("___PUBLIC_DSN___");
+    });
+  }
+}
+```
 
-        // Set the user in the current context.
-        context.setUser(new UserBuilder().setEmail("hello@sentry.io").build());
+### Send First Event
 
-        // This sends a simple event to Sentry.
-        sentry.sendMessage("This is a test");
+You can trigger your first event from your development environment by raising an exception somewhere within your application and sending it with `Sentry#captureException` method:
 
-        try {
-            unsafeMethod();
-        } catch (Exception e) {
-            // This sends an exception event to Sentry.
-            sentry.sendException(e);
-        }
-    }
+```java
+try {
+  System.out.println(1/0);
+} catch (Exception e) {
+  Sentry.captureException(e);
 }
 ```
+
+Once you've verified the library is initialized properly and sent a test event, consider visiting our [complete Java docs](https://docs.sentry.io/platforms/java/).
```

Comment:
should we just`/platforms/java/`? and let the engine create the redirect?

Reply:
The problem is that wizards are shown in different domain so I don't think it will create a correct absolute link to docs..?

Reply:
but then it'd not work here too: https://github.com/getsentry/sentry-docs/pull/2481/files#r501609138